src/utils/fetch.ts

```diff
@@ -0,0 +1,78 @@
+import queryStringify from './queryStringify';
+
+enum METHODS {
+  GET = 'GET',
+  POST = 'POST',
+  PUT = 'PUT',
+  DELETE = 'DELETE',
+}
+
+interface IOptions {
+  data?: any;
+  timeout?: number;
+  headers?: any;
```

Comment:
Можно типизировать детальнее, c data то же самое
```suggestion
  headers?: Record<string, string>;
```